tests/cxx/final_class_impl.h

```diff
@@ -0,0 +1,13 @@
+//===--- final_class_impl.h - test input file for iwyu --------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
```

Comment:
Please use include guards instead of `#pragma once`, see other tests for naming.